src/batch.rs

```diff
@@ -112,18 +113,18 @@ impl Item {
 
 /// A batch verification context.
 #[derive(Default)]
-pub struct Verifier {
+pub struct BatchVerifier {
```

Comment:
Is this renaming needed?

Reply:
I think there was some sort of conflict at one point. I'll have to go back and double check. Either way, I have a soft preference for it. Not a hill to die on but it is more accurate.

Reply:
My reasoning is that its full name is already `batch::Verifier` so `batch::BatchVerifier` is a bit redundant

Reply:
Fair enough. I'll put it back.